src/Shared/KnownFormats.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire;
+
+public static class KnownFormats
```

Comment:
Internal, in case we link to multiple projects and introduce conflicts.

```suggestion
internal static class KnownFormats
```